### ISSUE TITLE
Exclude example is badly formatted

### ISSUE THREAD

**Issue by johngeoffrey** [SamKirkland/FTP-Deploy-Action#126]
The example of "exclude" flag is not properly formatted for yaml and doesn't run.

Additionally once all line are created the following format (guessing this was the intended format), it is not expected by the exclude flag

``` 
exclude:
          - '.git*'
          - '.git*/**'
          - '**/.git*/**'
```

**Comment by SamKirkland** [comment]
Hello,

Can you post a screenshot of the error you are seeing?

The example in the readme is reported as valid yaml in my editor and is working in one of my test cases.
Your example above will not work due to the parsing limitation `@actions/core` - all inputs are returned as string and have to be [parsed manually on my side.](https://github.com/SamKirkland/FTP-Deploy-Action/blob/2d118d9190cfb7d3e55fae1a3ac6f2bcaca9e79c/src/parse.ts#L93) array support is finicky and needs improved.

**Comment by JackNUMBER** [comment]
I note the same point on VSCode, then I checked online with yamllint.com
![image](https://user-images.githubusercontent.com/2306550/100371497-23987e00-3008-11eb-995b-6019673bc3ba.png)

**Comment by adripo** [comment]
Please have a look at pr #133 for the solution

**Comment by SamKirkland** [comment]
Resolved in v4.1.0

**Comment by ewels** [comment]
@SamKirkland - apologies, but the examples in the readme still seem to suffer from the same problem. They give a YAML list instead of a string, and GitHub rejects them as shown in the screenshot above. This is true as of the v4.1.0 release.
